astropy/convolution/convolve.py

```diff
@@ -947,8 +947,7 @@ def convolve_fft(
     if crop:
         result = rifft[arrayslices].real
         return result
-    else:
-        return rifft.real
```

Comment:
Here we could replace the entire `if-else` block:
```python
    return rifft[arrayslices].real if crop else rifft.real
```
It would also be good to do this in other places where the return statement can be written as a oneliner.